Sources/LiveKit/Types/Options/CameraCaptureOptions.swift

```diff
@@ -19,9 +19,15 @@ import Foundation
 
 @objc
 public class CameraCaptureOptions: NSObject, VideoCaptureOptions {
+    /// Preferred deviceType to use. If ``device`` is specified, it will be used instead.
+    @objc
+    public let deviceType: AVCaptureDevice.DeviceType?
```

Comment:
@hiroshihorie this line doesn't build on visionOS. we need to add visionOs and tvOS to the CI matrix

Reply:
Ooh thanks, it's already added to the CI matrix. Wonder why it passes the test.

Reply:
Fixed the CI